webpack/mobile.config.js

```diff
@@ -0,0 +1,11 @@
+const { ProvidePlugin } = require('webpack')
+
+module.exports = {
+  plugins: [
+    new ProvidePlugin({
+      PouchDB: 'pouchdb',
+      pouchdbFind: 'pouchdb-find',
+      pouchdbAdapterCordovaSqlite: 'pouchdb-adapter-cordova-sqlite'
```

Comment:
Is it used somewhere?

Reply:
The adapter, yes, it's used in cozy-client-js. The 2 other variables I'm not sure…